build/validateNodeVersion.js

```diff
@@ -0,0 +1,112 @@
+const https = require('https');
+
+const SUPPORTED_LTS = "Carbon";
+const VERSION = process.versions.node;
+const NODE_JS_VERSION_LIST_URL="https://nodejs.org/dist/index.json"
+const MIN_MAJOR_SUPPORT = 8;
```

Comment:
Are we still relying solely off the information from nodejs.org? In that case why are these constants needed?